lib/rubocop/cop/layout/indentation_width.rb

```diff
@@ -383,6 +386,16 @@ def leftmost_modifier_of(node)
 
           leftmost_modifier_of(node.parent)
         end
+
+        def method_chain_block?(node)
+          send_node = node.send_node
+          receiver = send_node.receiver
+          return false unless receiver&.type?(:send, :block)
+          return false unless send_node.loc?(:dot) && send_node.loc.dot
```

Comment:
It looks like the tests pass even without this condition. Is it necessary, or is a test case missing?

```suggestion
```